pkg/githubutils/githubutils.go

```diff
@@ -0,0 +1,74 @@
+package githubutils
```

Comment:
I would rename this package, in general in Go it's discouraged to use names such as `utils` as they can be confusing. I would suggest either calling the package just `github`, or maybe or `githubapi`